packages/lib-user/src/components/GroupStats/EditGroup.js

```diff
@@ -4,42 +4,70 @@ import { useEffect, useState } from 'react'
 
 function EditGroup({ group, handleGroupUpdate }) {
```

Comment:
Noting the EditGroup component will be replaced with an iteration on the MyGroups/components/GroupForm, nonetheless it helps to have the following for now.